crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -533,6 +533,17 @@ module sui_system::sui_system {
         sui_system_state_inner::active_validator_addresses(self)
     }
 
+    /// Returns all the validators who are currently reporting `addr`
+    public fun get_reporters_of(wrapper: &mut SuiSystemState, addr: address): VecSet<address> {
```

Comment:
```suggestion
    public fun get_reporters_of(wrapper: &SuiSystemState, addr: address): VecSet<address> {
```

Reply:
Hello @sblackshear,

The function load_system_state requires a &mut SuiSystemState to load_inner_maybe_upgrade.
I believe this applies to all suggestions.